jina/serve/runtimes/worker/http_fastapi_app.py

```diff
@@ -130,20 +130,19 @@ def add_streaming_get_route(
 
         @app.api_route(
             path=f'/{endpoint_path.strip("/")}',
-            methods=['GET'],
+            methods=['POST'],
```

Comment:
I would rather duplicate the streaming endpoint in order to have 2 versions, post and get. The jina client should use the post method

Reply:
Sounds good to me, will add the get endpoint back.